### PR TITLE
[Feature] Implement 'Releases' Section in Blog Layout

### PR DESCRIPTION
**Pull request by mildrette**
fix: Implement Release section in blog layout,
- added no post available to the release and article section in case there are no post available at the moment
feat:  Adjusted the mobile menu layout to ensure proper alignment of items on smaller screens.

### CODE REVIEW

**Comment by PromiseFru** [comment]
Thank you, @mildrette!

Could you please provide a more descriptive title for your Pull Request?

There seems to be an issue with the `releases` tab, which disrupts the view when clicked.

image

Please take a look at it.

**Review by PromiseFru — CHANGES_REQUESTED** [review]
Thank you, @mildrette!

Could you please provide a more descriptive title for your Pull Request?

There seems to be an issue with the `releases` tab, which disrupts the view when clicked.

image

Please take a look at it.

**Comment by PromiseFru** [comment]
Thank you @mildrette!

There are a few more things to address:

- The alignment of the top bar has been distorted on mobile view.

image

- After creating a `_releases` directory, create a release post in the directory. When you click on the post from the release tab, it breaks the view.

image

File: src/app/_components/navbar.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ export function Navbar() {
 		<div className="border-b text-black border-neutral-300 bg-white w-full z-10 py-4 fixed top-0">
 			<Container>
 				<div className="flex justify-between items-center">
-					<Logo src="/icon.png" alt="SMSWithoutBorders Logo" name="SMSWithoutBorders" url="/" />
+					<Logo src="/icon.png" alt="SMSWithoutBorders Logo" name="SWOB" url="/" />
```
Comment (PromiseFru) [inline src/app/_components/navbar.tsx]: Why change the text here?

File: src/app/_components/navbar.tsx [hunk 2]
```diff
@@ -27,44 +27,92 @@ export function Navbar() {
...
+							Releases
+						</a>
 						<a
 							href="mailto:support@smswithoutborders.com"
-							target="_blank"
-							rel="noopener noreferrer"
+							className="text-gray-700 hover:text-blue-500 transition duration-300"
 						>
-							<FaEnvelope size={20} className="rounded" />
+							<FaEnvelope size={30} className="rounded bg-white p-1" />
```
Comment (PromiseFru) [inline src/app/_components/navbar.tsx:45]: Why remove the link `target`?

File: src/app/_components/navbar.tsx [hunk 3]
```diff
@@ -27,44 +27,92 @@ export function Navbar() {
...
+						>
+							<FaGithub size={30} className="rounded-full bg-white p-1" />
 						</a>
-						<a href={LINKEDIN_URL} target="_blank" rel="noopener noreferrer">
-							<FaLinkedin size={20} className="rounded" />
+						<a
+							href={LINKEDIN_URL}
+							className="text-gray-700 hover:text-blue-500 transition duration-300"
+						>
+							<FaLinkedin size={30} className="rounded-full bg-white p-1" />
```
Comment (PromiseFru) [inline src/app/_components/navbar.tsx]: We should ensure that all items have a uniform style, including text size, colours, and effects.

File: src/app/_components/post-preview.tsx [hunk 4]
```diff
@@ -42,3 +42,37 @@ export function PostPreview({ title, date, excerpt, author, slug, content }: Pro
...
 }
+
+export function ReleasePreview({ title, date, excerpt, author, slug, content }: Props) {
+	return (
+		<div className="border border-neutral-300 rounded-lg bg-white py-20 px-5 md:px-24 gap-y-5 mb-20">
+			<h1 className="text-2xl md:text-3xl mb-3 mx-4 text-center">
+				<Link
+					as={`/releases/${slug}`}
+					href="/releases/[slug]"
+					className="hover:unde/postsrline font-semibold"
```
Comment (PromiseFru) [inline src/app/_components/post-preview.tsx]:
```suggestion
					className="hover:underline font-semibold"
```

File: src/app/_components/navbar.tsx
Comment (PromiseFru) [inline src/app/_components/navbar.tsx:1]: We should arrange the icons horizontally for the mobile view.

image

**Review by PromiseFru — CHANGES_REQUESTED** [review]
Thank you, @mildrette! Just a few more tweaks 😉

File: src/app/_components/navbar.tsx
Comment (mildrette) [reply src/app/_components/navbar.tsx:1]: all done

**Review by PromiseFru — DISMISSED** [review]
Thank you, @mildrette!

**Pull request merged**